script/Ciel/Day 8/08_01_CIEL08_2.txt

```diff
@@ -128,30 +143,36 @@ She closes the curtains surrounding the bed.
 [sha:7f4f8c0c9711e86fbc15dfd68db8e1de6a24f208]{
 -- Page 165, Offset 18690.
 -- ノエル先生は陽気に口笛などを吹きつつ横になってしまった。
-Noel-sensei lies down on the bed while merrily whistling a tune.
+Noel-sensei lies down bed while merrily whistling a tune.
 }
 [sha:a18de3ce03d78e31ef10e09f6c0cd0d17a47181a]{
 -- Page 166, Offset 18691.
 -- 「あの。俺が言える事じゃないけど、教師がそんな簡単に休んでいいんですか？」
-"Um. I know I'm not in the position to say this, but is it really okay for a teacher to be skipping work like that?"
+"Um... I know it's not my place to say, but is it really alright for a teacher to up and skip work like this?"
 }
 [sha:9b7fb23c7f632e5dbd0bd78ea99d376f693a5475]{
 -- Page 166, Offset 18692.
 -- 「ホント、キミが言えるコトじゃないわね。オトナだってさぼりたい時があるのよ。いいじゃない、自習の一つや二つ。わたしサンタクロースみたいなもんだし。ちょっと早いけど、子供たちにプレゼントをあげましょう！」
-"Yup, that's really not something for you to say. Even adults want to slack off sometimes. One or two sessions of self-study, that's not a big deal. I'll be like Santa Claus or something, giving presents to children! A bit early, I admit."
+"Yep, it's not your place to say that at all.
+ Even adults want to slack off sometimes.
+ C'mon, what's the harm in one or two lessons being self-study?
+ I'm just like Santa Claus, delivering presents to the children!
+ A bit early, I admit."
 }
 [sha:013786638a8d9093f503aa141c57b50c708091b6]{
 -- Page 167, Offset 18693.
 -- 妙な言い訳をして彼女は眠ってしまった。
-And with that strange excuse, she goes off to sleep.
+After giving that bizarre excuse, she falls asleep.
 }
 [sha:0765ca21886dbdd81f3b5ba834594aae0f8aca1a]{
 -- Page 167, Offset 18694.
 -- 起こすのも気が引けるし、こっちも急がないと４時限目に間に合わない。
-I'm reluctant to wake her up, and besides, I have to hurry up or I won't make it for fourth period.
+I'd feel awkward about waking her up,
+ and I've got to get going anyway if I want to make it in time for fourth period.
 }
 [sha:4f6125cef3d6f3fa6e7eaf4144fb195917cdc6a2]{
 -- Page 167, Offset 18695.
 -- 一応、薬箱から風邪薬を出してテーブルに置いてから、保健室を後にした。
-Just in case, I take out some cold medication from the medicine box, leaving it on the table, and then leave the infirmary.
+I take some cold medicine from the medicine cabinet and place it on the table just in case,
```

Comment:
Maybe shorten cold medicine to 'cold meds' to lessen the repetition?

Reply:
ehhhhh i dunno about that. I think the current version is fine but i see where you're coming from. If you want to reduce the repetition it should really just become "cabinet" and leave "cold medicine" alone